json-schema/json-schema/src/main/scala/endpoints/algebra/Codec.scala

```diff
@@ -68,4 +67,40 @@ object Codec {
       Decoder.sequentially(ab)(bc)
     )
 
+  /** Produce a codec that passes values through without touching them */
+  def identity[A]: Codec[A, A] = new Codec[A, A] {
```

Comment:
Is this one used somewhere?

Reply:
No, it isn't. I figured it might start being useful now that `Codec` can be used as an argument in more places and that the cost of including it is relatively low.

Motivating example: in my companies codebase, we have a trait somewhat like:

```scala
trait CompExt extends algebra.Endpoints {

  /** The decoded type of binary data */
  type BStr

  /** Since binary data gets encoded as Base64 strings, it is necessary to
    * define the serialization/deserialization to/from byte arrays
    */
  def byteStringCodec: endpoints.algebra.Codec[Array[Byte], BStr]

  /** Schema for binary data encoding (uses base64 encoded strings) */
  implicit lazy val byteStringSchema: JsonSchema[BStr] = {
    val enc = java.util.Base64.getEncoder
    val dec = java.util.Base64.getDecoder

    stringJsonSchema(format = Some("byte"))
      .withExample("abcdef")
      .xmapPartial(tryParseString("base64 string")(dec.decode(_)))(enc.encodeToString(_))
      .xmapWithCodec(byteStringCodec)
  }
}
```

Then, the Scala backend may implement this with `type BStr = akka.util.ByteString` while the Scala.js front-end (which doesn't have an Akka dependency) sticks with `type BStr = Array[Byte]`. Then it is handy to be able to do `val byteStringCodec = Codec.identity[Array[Byte]]` in the client.

Shall I take it out for now?

Reply:
I see that you already removed it. Fine, we can re-introduce it if you feel it should be there :)